dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -59,6 +65,12 @@
   public static final RemoteRepository CENTRAL =
       new RemoteRepository.Builder("central", "default", "http://repo1.maven.org/maven2/").build();
 
+  private static ImmutableList<RemoteRepository> mavenRepositories = ImmutableList.of(CENTRAL);
```

Comment:
Mutable static state suggests we're growing past the point where this class can plausibly be static. OK, for now but consider whether refactoring might help. Either to use more instance methods or to pass the repsositoryURLs as arguments to a method instead.

Reply:
Created #353